UefiPayloadPkg/Tools/MkFitImage.py

```diff
@@ -76,24 +76,24 @@ def BuildTianoImageNode(Fdt, InfoHeader, ParentNode, DataOffset, DataSize, Descr
     if InfoHeader.LoadAddr is not None:
         libfdt.fdt_setprop_u64(Fdt, ParentNode, 'load', InfoHeader.LoadAddr)
     if InfoHeader.Entry is not None:
-        libfdt.fdt_setprop_u64(Fdt, ParentNode, 'entry-start', InfoHeader.Entry)
+        libfdt.fdt_setprop_u64(Fdt, ParentNode, 'entry', InfoHeader.Entry)
     if InfoHeader.RelocStart is not None:
         libfdt.fdt_setprop_u32(Fdt, ParentNode, 'reloc-start', InfoHeader.RelocStart)
     if InfoHeader.DataSize is not None:
        libfdt.fdt_setprop_u32(Fdt, ParentNode, 'data-size', DataSize)
     if InfoHeader.DataOffset is not None:
         libfdt.fdt_setprop_u32(Fdt, ParentNode, 'data-offset', DataOffset)
     if InfoHeader.Producer is not None:
-        libfdt.fdt_setprop(Fdt, ParentNode, 'producer ', bytes(InfoHeader.Producer, 'utf-8'), len(InfoHeader.Producer) + 1)
+        libfdt.fdt_setprop(Fdt, ParentNode, 'producer', bytes(InfoHeader.Producer, 'utf-8'), len(InfoHeader.Producer) + 1)
     if InfoHeader.Capabilities is not None:
         CapStrs = ','.join(InfoHeader.Capabilities)
-        libfdt.fdt_setprop(Fdt, ParentNode, 'capabilities ', bytes(CapStrs, 'utf-8'), len(CapStrs) + 1)
+        libfdt.fdt_setprop(Fdt, ParentNode, 'capabilities', bytes(CapStrs, 'utf-8'), len(CapStrs) + 1)
     if InfoHeader.Type is not None:
-        libfdt.fdt_setprop(Fdt, ParentNode, 'type ', bytes(InfoHeader.Type, 'utf-8'), len(InfoHeader.Type) + 1)
+        libfdt.fdt_setprop(Fdt, ParentNode, 'type', bytes(InfoHeader.Type, 'utf-8'), len(InfoHeader.Type) + 1)
     if InfoHeader.Arch is not None:
-        libfdt.fdt_setprop(Fdt, ParentNode, 'arch ', bytes(InfoHeader.Arch, 'utf-8'), len(InfoHeader.Arch) + 1)
+        libfdt.fdt_setprop(Fdt, ParentNode, 'arch', bytes(InfoHeader.Arch, 'utf-8'), len(InfoHeader.Arch) + 1)
     if InfoHeader.Project is not None:
-        libfdt.fdt_setprop(Fdt, ParentNode, 'project ', bytes(InfoHeader.Project, 'utf-8'), len(InfoHeader.Project) + 1)
+        libfdt.fdt_setprop(Fdt, ParentNode, 'project', bytes(InfoHeader.Project, 'utf-8'), len(InfoHeader.Project) + 1)

```

Comment:
with above changes , GetUplVerFromBin.py also needs to update, otherwise will cause build error.

Reply:
I don't see this file in either the edk2 or edk2-platforms repos, could you please point me to its location? If this is a script you use downstream/in closed-source code, you're welcome to make the changes.

I can mark this as a breaking change though: I hadn't before because I had seen that all EDK2 code used `AsciiStrnCmp` without the final space character

Reply:
Sorry , my mistake , this file is in intel repo. Please ignore it , I will update the other PR for it.